src/index.html

```diff
@@ -17,6 +17,13 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
```

Comment:
you should add this modifier here, not for hands directly
![image](https://github.com/mate-academy/layout_stop-watch/assets/55200530/6e048982-4931-4834-be81-0ed11494a4d3)
```suggestion
    <div class="stopwatch stopwatch--speed-up">
```